website/components/Typeform.tsx

```diff
@@ -0,0 +1,19 @@
+import { Widget } from '@typeform/embed-react';
+
+export const Typeform = ({
+                             id,
+                         }: {
+    id: string;
+}) => {
```

Comment:
IDK if this weird formatting a norm in JS
```suggestion
export const Typeform = ({id}: {id: string}) => {
```